api/src/main/java/org/eclipse/microprofile/health/spi/SPIFactory.java

```diff
@@ -20,22 +20,15 @@
  *
  */
 
-package org.eclipse.microprofile.health;
+package org.eclipse.microprofile.health.spi;
 
-import java.util.Map;
-import java.util.Optional;
+import org.eclipse.microprofile.health.ResponseBuilder;
 
 /**
- * The health procedure response
+ * Created by hbraun on 07.07.17.
  */
-public interface HealthStatus {
+public interface SPIFactory {
```

Comment:
Name is a generic and non-descriptive. Maybe something like `ResponseBuilderFactory` instead?